hnn_core/tests/test_compare_hnn.py

```diff
@@ -87,6 +92,12 @@ def test_compare_across_backends():
     # test consistency between mpi backend simulation (n_procs=2) and master
     dpls_reduced_mpi = run_hnn_core(backend='mpi')
 
+    try:
+        import mpi4py
+        mpi4py.__file__
+    except ImportError:
+        pytest.skip("mpi4py not available")
```

Comment:
does the test in L93 make sense if mpi4py is not available?

Reply:
Nope. I put it on the wrong line...

Reply:
I updated this PR to stop using pytest.skip, but skip running and comparing MPI simulations. The joblib tests should still complete and return a pass/fail.